docs/reference/values.md

```diff
@@ -10,11 +10,12 @@ You can read and write [HTML data attributes](https://developer.mozilla.org/en-U
 <meta data-controller="callout" data-callout-text-value="data-loader-url-value=&quot;/messages&quot;">
 
 ```html
-<div data-controller="loader"
-     data-loader-url-value="/messages">
+<div data-controller="loader" data-loader-url-value="/messages">
 </div>
 ```
 
+Data attributes used for values must be declared on the controlled element, the same one that has the `data-controller` attribute.
```

Comment:
I think we are mostly referring the element which holds the `data-controller`  attribute as the "controller element":
```suggestion
Data attributes used for values must be declared on the controller element, the same one that has the `data-controller` attribute.
```

Reply:
I believe we can omit the term `API` since the whole page is about Values. I will think we have enough context.

Reply:
It's weird to have multiple different terms for the same thing though, especially if there is already an established one.

Reply:
I undestand. I change the text to a better one to include ideas from you comment and the previous one.